tests/unit/conftest.py

```diff
@@ -174,6 +174,19 @@ def start_cmd_handler(argv: list[str]) -> synapse.ExecResult:
         executable="/usr/bin/python3",
         handler=lambda _: synapse.ExecResult(0, "", ""),
     )
+    synapse_nginx_container: ops.Container = harness.model.unit.get_container(
+        synapse.SYNAPSE_NGINX_CONTAINER_NAME
+    )
+    harness.register_command_handler(  # type: ignore # pylint: disable=no-member
```

Comment:
Out of curiosity, may I ask why the pylint disable is applied here? :D

Reply:
Sorry, I didnt add the comment but the reason is to ignore this error "error: "Harness[SynapseCharm]" has no attribute "register_command_handler"  [attr-defined]"

Reply:
It will be addressed in https://github.com/canonical/synapse-operator/issues/275